src/domains/idpay-common/03_database.tf

```diff
@@ -75,4 +75,245 @@ module "mongdb_collection_onboarding_citizen" {
   ]
 
   lock_enable = false #TODO temporary only for dev
-}
\ No newline at end of file
+}
+
+# Collections
```

Comment:
Can we aggregate all this collections and have only one list to iterate?

Reply:
As requested, it has been handled as a list